app/dash/Chains/index.js

```diff
@@ -87,10 +87,25 @@ class Settings extends React.Component {
               const key = type + id
               const { explorer, isTestnet, connection, on, name } = networks[type][id]
               const {
-                nativeCurrency: { symbol = '?' }
+                nativeCurrency: { symbol = '?', name: nativeCurrencyName, icon: nativeCurrencyIcon },
+                icon
               } = metadata[type][id]
-              const chain = { id, type, symbol, explorer, isTestnet, connection, on, filter, name }
-              return <Chain key={key} {...chain} view={'preview'} />
+              const chain = {
+                key,
+                id,
+                type,
+                symbol,
+                explorer,
+                isTestnet,
+                connection,
+                on,
+                filter,
+                name,
+                nativeCurrencyName,
+                nativeCurrencyIcon,
+                icon
+              }
+              return <Chain {...chain} view={'preview'} />
```

Comment:
```suggestion
              return <Chain key={key} {...chain} view={'preview'} />
```

And then remove key from Chain above - key is logically separate and you won't get a linting error this way.